app/src/main/java/net/sourceforge/opencamera/ui/MainUI.java

```diff
@@ -929,7 +930,19 @@ public void setTakePhotoIcon() {
             int resource;
             int content_description;
             int switch_video_content_description;
-            if( main_activity.getPreview().isVideo() ) {
+            boolean is_client;
+            {
```

Comment:
Why do we need brackets here?

Reply:
So that `appInterface` is not visible outside this block - we don't need it later in the function